collectorist/README.md

```diff
@@ -0,0 +1,44 @@
+# A Vertical Slice 
+
+## GUAC
+
+In some window, run GUAC according to the upstream GUAC instructions:
+
+    docker compose up --force-recreate
+
+Leave that process running.
+
+## Collectorist
+
+The Collectorist drives the polling and populating of GUAC.
+Run a single instance of the Collectorist as we do, using the `trust` CLI after building this repository.
+
+    RUST_LOG=info ./target/debug/trust collectorist api
+
+## OSV Collector
+
+The OSV collector is driven by the Collectorist. 
+It must be run with the collectorist URL specified, which defaults to the correct URL unless you're getting weird.
+
+    RUST_LOG=info ./target/debug/trust collectorist api
+
+# What's going on?
```

Comment:
Thanks for the README. How do you trigger something to happen? Do you fire up the sbom/vex walkers? Or feed some OSV data file? Something else?